modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManagementInformationMBean.java

```diff
@@ -19,25 +19,47 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
 import org.wso2.andes.kernel.AndesContext;
 import org.wso2.andes.kernel.AndesException;
 import org.wso2.andes.management.common.mbeans.ClusterManagementInformation;
 import org.wso2.andes.management.common.mbeans.annotations.MBeanConstructor;
 import org.wso2.andes.server.management.AMQManagedObject;
 
-import java.util.List;
 import javax.management.JMException;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
 
 /**
  * <code>ClusterManagementInformationMBean</code> The the JMS MBean that expose cluster management information
  * Exposes the Cluster Management related information using MBeans
  */
 public class ClusterManagementInformationMBean extends AMQManagedObject implements ClusterManagementInformation {
+
     /**
      * Class logger
      */
     private static final Log logger = LogFactory.getLog(ClusterManagementInformationMBean.class);
-
+    private final String IP_LIST = "IpList";
+    private final String NODE = "Node";
+    private final String ADDRESSES = "Addresses";
```

Comment:
Please differentiate between "IP_LIST" and "ADDRESSES" when you describe the variables